src/components/RollupProductDevDoc.js

```diff
@@ -0,0 +1,65 @@
+// Libraries
+import React from "react"
+import styled from "styled-components"
+
+// Components
+import Link from "./Link"
+
+// Data
+import optimisticRollupList from "../data/layer-2-data-files/optimistic-rollups.json"
+import zkRollupList from "../data/layer-2-data-files/zk-rollups.json"
```

Comment:
Discussed at stand-up the idea of combining these into a single data file under `../data/layer-2.json` with root level keys `"optimistic-rollups"` and `"zk-rollups"`.
@corwintines curious your thought on this, if you were partial to keeping them separate for any reason?

Reply:
Will combine, definitely open. I don't have a strong opinion for keeping them separate, more just mental map stuff but we can break out if these lists get too large.